BuyPolish/Pola/AppDelegate.swift

```diff
@@ -29,6 +31,16 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             _ = handleShortcutItem(shortcutItem)
         }
 
+        UNUserNotificationCenter.current().delegate = self
```

Comment:
Co sądzisz o wyniesieniu tego od oddzielnej klasy? Np. NotificationHandler/NotificationManager, tak żeby w AppDelegate było tylko wywołanie metody `register(in application)`, a nowa klasa implementowałą też delegata.